client/src/components/NearestHospital/LocalEmergencyNumber.js

```diff
@@ -67,9 +68,11 @@ const EmergencyLine = styled(NoticeBar)`
 `;
 
 const LocalEmergencyNumber = (props) => {
+  const { t } = useTranslation();
+
   return (
     <EmergencyLine mode="closable" icon={null} {...props}>
-      <h4>Local emergency number</h4>
+      <h4>{t("nearestHsp.locaEmrgNbr")}</h4>
```

Comment:
locaEmrgNbr to localEmergencyNumber?

Reply:
This component will be developed in the future with another PR/feature branch.  (probably by me in the near future).

Reply:
done